application/healthcare-services-data-manager/app.py

```diff
@@ -1,15 +1,23 @@
+import os
 import service
 from aws_lambda_powertools.event_handler import APIGatewayRestResolver
 from aws_lambda_powertools.utilities.typing import LambdaContext
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.logging import correlation_paths
 
 app = APIGatewayRestResolver()
 
-
 # Auto resolves the type of request comming through and sets APIGatewayRestResolver
 # fields
 
+log_level = os.environ.get("LOG_LEVEL", "info")
+logger = Logger(service="healthcare_services", level=log_level)
+
 
+@logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST)
 def lambda_handler(event: dict, context: LambdaContext) -> dict:
+    logger.info(event)
```

Comment:
Do we need both lines ? Either logger.info or logger.debug would suffice no ?
Same applies to the other data manager lambdas

Reply:
Both are required otherwise only the one stated will be reported..have tested several times